src/fmu/dataio/datastructure/meta/meta.py

```diff
@@ -186,71 +197,113 @@ class RealizationJobListing(BaseModel):
 
 
 class Realization(BaseModel):
-    id: int = Field(
-        description="The unique number of this realization as used in FMU",
-    )
-    name: str = Field(
-        description="The convential name of this iteration, e.g. iter-0 or pred",
-        examples=["iter-0"],
-    )
-    parameters: Optional[Parameters] = Field(
-        default=None,
-        description="Parameters for this realization",
-    )
-    jobs: Optional[object] = Field(
-        default=None,
-        description=(
-            "Content directly taken from the ERT jobs.json file for this realization"
-        ),
-    )
+    """The ``fmu.realization`` block contains information about the realization this
+    data object belongs to."""
+
+    id: int
+    """The internal ID of the realization, typically represented by an integer."""
+
+    name: str = Field(examples=["iter-0"])
+    """The name of the realization. This is typically reflecting the folder name on
+    scratch. Custom names for realizations are not supported by ERT, but we still
```

Comment:
Instead of "Custom names for realizations..." perhaps:

"We recommend to use `fmu.realization.uuid` for all usage except purely visual appearance."

Guiding consumers to the `uuid`, which we control, feels safer?